gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -103,13 +103,15 @@ class Connection extends EventEmitter {
     this._pingIntervalDelay = this.options.pingInterval || pingIntervalDelay;
     this._pongTimeoutDelay = this.options.pongTimeout || pongTimeoutDelay;
 
-    if (this.options.connectOnStartup !== false) {
-      this.open();
+    if (this.options.connectOnStartup) {
```

Comment:
The above comment that was updated used to say the default was true, but that is not entirely correct is it? Technically it was just `!== false`?

Just want to ensure this console warning will not be printing out when a user does not set this issue.